scripts/animatediff.py

```diff
@@ -217,6 +217,16 @@ def on_ui_settings():
             section=section
         )
     )
+    shared.opts.add_option(
+        "animatediff_default_save_formats",
+        shared.OptionInfo(
+            ["GIF", "PNG"],
+            "Default Save Formats",
+            gr.CheckboxGroup,
+            {"choices": supportedSaveFormats},
+            section=section
+        )
+    )
```

Comment:
I had to restart A1111 for these settings to apply. Might be good to add a note that restart is required:
```
.info("restart required")
```